<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_clock">
       <property name="styleSheet">
        <string notr="true">#page_clock{border-image: url(:/tomato/png/image/layer1-5.png);}</string>
       </property>
       <widget class="QLabel" name="label_time">
        <property name="geometry">
         <rect>
          <x>189</x>
          <y>106</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>186</x>
          <y>130</y>
          <width>237</width>
          <height>65</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLCDNumber" name="lcd_time">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="digitCount">
            <number>8</number>
           </property>
           <property name="mode">
            <enum>QLCDNumber::Dec</enum>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="value" stdset="0">
            <double>888888.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>888888</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>140</y>
          <width>32</width>
          <height>34</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_am">
           <property name="font">
            <font>
             <family>黑体</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>上午</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_pm">
           <property name="font">
            <font>
             <family>黑体</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>下午</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>220</y>
          <width>440</width>
          <height>16</height>
         </rect>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>229</y>
          <width>3</width>
          <height>70</height>
         </rect>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>229</y>
          <width>3</width>
          <height>70</height>
         </rect>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>229</y>
          <width>3</width>
          <height>70</height>
         </rect>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_yangli">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>240</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="text">
         <string>阳历</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_nongli">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>270</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>农历</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>233</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLCDNumber" name="lcd_date">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="intValue" stdset="0">
            <number>88</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_lunardate">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>270</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>280</y>
          <width>76</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLCDNumber" name="lcd_lunarmonth">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="intValue" stdset="0">
            <number>88</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcd_lunarday">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="intValue" stdset="0">
            <number>88</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_wday">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>230</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>星期</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_wday">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>240</y>
          <width>41</width>
          <height>51</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="digitCount">
         <number>1</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>8</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_temperature">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>241</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>摄氏度</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_shidu">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>269</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>湿度</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_temperature">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>233</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>88</number>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcd_shidu">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>262</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>88</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonMemo">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>380</y>
          <width>79</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>备注</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonHide">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>380</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>隐藏</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonClose">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>380</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_tomato">
       <property name="styleSheet">
        <string notr="true">#page_tomato{border-image: url(:/tomato/png/image/layer1-5.png);}</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionS">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
